products/BellHybrid/services/db/databases/migration/settings_bell/current/1b034c28_Change_tone_names_to_tone_paths/.meta

```diff
@@ -0,0 +1,6 @@
+{
+ "id": "1b034c28-f8f4-4792-8f35-9ab376fec0de",
+ "date": "2024-03-06 15:07:49",
+ "message": "Change tone names to tone paths",
+ "parent": 0
+}
```

Comment:
I've seen that but TBH I'd prefer to not modify this file manually - IIRC it will be removed after the migration is committed before release and as it is autogenerated I'm not sure how the script operating on it will react after such manual change